BLOG.md

```diff
@@ -0,0 +1,996 @@
+
+#### 3. Fulfill the live shopping request
+
+Let's add a custom `action` to assign a support user to the live shopping request.
+
+```python
+    @action(methods=["post"], detail=True)
+    def start(self, request, pk=None):
+        live_request = get_object_or_404(LiveVideoRequest, pk=pk)
+        if live_request.support_user_dyte_participant_id:
+            token = DyteAPIClient.refresh_participant_token(
+                live_request.dyte_meeting_id,
+                live_request.support_user_dyte_participant_id,
+            )
+            return Response({"dyte_auth_token": token["token"]})
+        participant = DyteAPIClient.add_participant(
+            live_request.dyte_meeting_id,
+            "Customer Support",
+            "video_shoping",
+            "customer_support",
+        )
+        live_request.support_user_dyte_participant_id = participant["id"]
+        live_request.status = LiveVideoRequest.ACTIVE
+        live_request.save()
+        return Response(
+            {"dyte_auth_token": participant["token"]}, status=status.HTTP_201_CREATED
+        )
+```
+We are using the `DyteAPIClient.add_participant` to add a `Customer Support` participant to our Dyte meeting and returns `dyte_auth_token` for this participant.
+
+#### 4. Getting dyte participant token for user
+
+Now, let's add another custom action, `user_token`, to retrieve `dyte_auth_token` for the customer. We will use this API to instantiate DyteClient for user.
+
+```python
+    @action(methods=["get"], detail=True, url_path="user-token")
+    def user_token(self, request, pk=None):
+        live_request = get_object_or_404(LiveVideoRequest, pk=pk)
+        token = DyteAPIClient.refresh_participant_token(
+            live_request.dyte_meeting_id, live_request.user_dyte_participant_id
+        )
+        return Response({"dyte_auth_token": token["token"]})
+```
+
+Refer [DRF docs](https://www.django-rest-framework.org/) to learn more about adding authentication and permission to the ViewSet.
+
+
+### Creating Router
+
+Let's now register a router for `LiveVideoRequestViewSet`. Create a new file `urls.py` in the `live_shopping` folder and add the router.
+
+```python
+from rest_framework.routers import SimpleRouter
+
+from .views import LiveVideoRequestViewSet
+
+router = SimpleRouter()
+router.register(r"live-requests", LiveVideoRequestViewSet, basename="live_requests")
+
+urlpatterns = router.urls
+```
+
+Now update the `live_shopping_backend/urls.py` to include the API endpoints.
+
+```python
+from django.urls import path, include
+
+...
+urlpatterns = [
+	...
+	path("live-shopping/", include("live_shopping.urls")),
+	...
+]
+```
+
+
+### Starting the development server.
+
+Once we have set up your Django project and app, we can start the development server to test your code. To start the Django dev server, open a terminal and navigate to the root directory of our Django project. Then, enter the following command:
```

Comment:
either write 'we' and 'our' or 'you' and 'your' in one sentence - so either 'Once _you_ have set up your Django project and app' or 'Once we have set up _our_ Django project and app'